### ISSUE TITLE
Using 'order' property breaks styled-jsx

### ISSUE THREAD

**Issue by benhjames** [zeit/styled-jsx#232]
Using the `order` CSS property seems to break updating from `styled-jsx@0.5.3` to `styled-jsx@1.0.3`:

```jsx
<style jsx>{`
  div {
    order: 1;
  }
`}</style>
```

Results in:

```
Module build failed: ReferenceError: /...redacted.../hocs/defaultLayout.js: flex is not defined
    at property (/...redacted.../node_modules/stylis/stylis.js:806:32)
    at compile (/...redacted.../node_modules/stylis/stylis.js:358:16)
    at compile (/...redacted.../node_modules/stylis/stylis.js:313:17)
    at stylis (/...redacted.../node_modules/stylis/stylis.js:1076:16)
    at transform (/...redacted.../node_modules/styled-jsx/dist/lib/style-transform.js:100:10)
    at PluginPass.exit (/...redacted.../node_modules/styled-jsx/dist/babel.js:261:85)
    at newFn (/...redacted.../node_modules/babel-traverse/lib/visitors.js:276:21)
    at NodePath._call (/...redacted.../node_modules/babel-traverse/lib/path/context.js:76:18)
    at NodePath.call (/...redacted.../node_modules/babel-traverse/lib/path/context.js:48:17)
    at NodePath.visit (/...redacted.../node_modules/babel-traverse/lib/path/context.js:117:8)
```

Removing the order property makes it work again.

Let me know if there's anything else you need!

**Comment by benhjames** [comment]
It seems this was fixed in `stylis@3.0.15`, which is used in `styled-jsx@1.0.4`, but the `next` version I'm using (2.4.3) is currently pointing at `styled-jsx@1.0.3`. It just needs bumping at `zeit/next`!

**Comment by giuseppeg** [comment]
sorry for that, I will try to sync up with the Next.js folks to see if we can get styled-jsx fixes / version bumps  out much quickly
